docs/source/topics/prices_and_availability.rst

```diff
@@ -280,6 +282,7 @@ Here's an example ``strategy.py`` module which is used to charge VAT on prices.
     # myproject/partner/strategy.py
 
     from oscar.apps.partner import strategy, prices
+    from decimal import Decimal as D
```

Comment:
We should use PEP8 import order - i.e., this should come above the `oscar` import.